recipes/wildmeshing/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "wildmeshing" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wildmeshing/wildmeshing-python/archive/{{ version }}.tar.gz
+  sha256: 12c85ff3ec97d7020bb705dcc76f4a545be13c5537b78ae66edbf4749a6ee56c
+
+build:
+  number: 0
+  skip: True  # [py<31]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - python
+    - pip
+    - libblas
+    - libcblas
+    - numpy
+    - gmp     # [unix]
+    - mpir    # [win]
+  run:
+    - python
+    - mpir    # [win]
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - wildmeshing
+
+about:
+  home: https://github.com/wildmeshing/wildmeshing-python
+  license: LGPL
+  license_family: LGPL
```

Comment:
Depending on the outcome of above, this might need to be changed to `MPL`.

Reply:
Ok i can change it.

It was `MPL-2.0` and someone asked me to change it
https://github.com/conda-forge/staged-recipes/pull/8525/commits/e883cc0d215959178ba66ee22236839219b4a711

can you plz agree, I have the feeling this is not going anywhere

Reply:
also the family must be one of these:
`AGPL,APACHE, BSD, CC, GPL, GPL2, GPL3, LGPL, MIT, NONE, OTHER, PROPRIETARY,
PSF, and PUBLIC-DOMAIN`

this is why I had MIT, which i belive is the closest one

Reply:
It should be LGPL because there's a LGPL component vendored.